draft-ietf-tls-tls13.md

```diff
@@ -3872,91 +3885,35 @@ and renegotiation_info {{RFC5746}}.
-| TLS_DHE_RSA_WITH_AES_256_CCM                  | {0xC0,0x9F} | [RFC6655]     |
-| TLS_DHE_RSA_WITH_AES_128_CCM_8                | {0xC0,0xA2} | [RFC6655]     |
-| TLS_DHE_RSA_WITH_AES_256_CCM_8                | {0xC0,0xA3} | [RFC6655]     |
-| TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xA8}   | [RFC7905] |
-| TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256 | {0xCC,0xA9}   | [RFC7905] |
-| TLS_DHE_RSA_WITH_CHACHA20_POLY1305_SHA256     | {0xCC,0xAA}   | [RFC7905] |
-
-Note: ECDHE AES GCM was not yet standards track prior to the publication of
-this specification. This document promotes the above-listed ciphers to
-standards track.
-
-The following is a list of standards track ephemeral pre-shared key cipher
-suites which are currently available in TLS 1.3:
-
-|              Cipher Suite Name                |    Value    | Specification |
-|:----------------------------------------------|:------------|:--------------|
-| TLS_DHE_PSK_WITH_AES_128_GCM_SHA256           | {0x00,0xAA} | [RFC5487]     |
-| TLS_DHE_PSK_WITH_AES_256_GCM_SHA384           | {0x00,0xAB} | [RFC5487]     |
-| TLS_DHE_PSK_WITH_AES_128_CCM                  | {0xC0,0xA6} | [RFC6655]     |
-| TLS_DHE_PSK_WITH_AES_256_CCM                  | {0xC0,0xA7} | [RFC6655]     |
-| TLS_PSK_DHE_WITH_AES_128_CCM_8                | {0xC0,0xAA} | [RFC6655]     |
-| TLS_PSK_DHE_WITH_AES_256_CCM_8                | {0xC0,0xAB} | [RFC6655]     |
-| TLS_ECDHE_PSK_WITH_AES_128_GCM_SHA256         | {0xD0,0x01}   | [I-D.mattsson-tls-ecdhe-psk-aead] |
-| TLS_ECDHE_PSK_WITH_AES_256_GCM_SHA384         | {0xD0,0x02}   | [I-D.mattsson-tls-ecdhe-psk-aead] |
-| TLS_ECDHE_PSK_WITH_AES_128_CCM_8_SHA256       | {0xD0,0x03}   | [I-D.mattsson-tls-ecdhe-psk-aead] |
-| TLS_ECDHE_PSK_WITH_AES_128_CCM_SHA256         | {0xD0,0x04}   | [I-D.mattsson-tls-ecdhe-psk-aead] |
-| TLS_ECDHE_PSK_WITH_AES_256_CCM_SHA384         | {0xD0,0x05}   | [I-D.mattsson-tls-ecdhe-psk-aead] |
-| TLS_ECDHE_PSK_WITH_CHACHA20_POLY1305_SHA256   | {0xCC,0xAC}   | [RFC7905] |
-| TLS_DHE_PSK_WITH_CHACHA20_POLY1305_SHA256     | {0xCC,0xAD}   | [RFC7905] |
-
-Note: The values listed for ECDHE PSK AES suites are preliminary but
-are being or will be used for interop testing and therefore are likely to be
-assigned.
-
-Note: [RFC6655] is inconsistent with respect to the ordering of components
-within PSK AES CCM cipher suite names. The names above are as defined.
-
-All cipher suites in this section are specified for use with both TLS 1.2
-and TLS 1.3, as well as the corresponding versions of DTLS.
-(see {{backward-compatibility}})
+represents 256-bit AES in the GCM mode of operation. 
+
+|       Cipher Suite Name         |    Value    | Specification |
+|:--------------------------------|:------------|:--------------|
+| TLS_AES_128_GCM_SHA256        | {0x13,0x01} |   [This RFC]  |
+| TLS_AES_256_GCM_SHA384        | {0x13,0x02} |   [This RFC]  |
+| TLS_CHACHA20_POLY1305_SHA256  | {0x13,0x03} |   [This RFC]  |
```

Comment:
>We're no more banning it than we are banning CAMELLIA, which also can in
 >principle be used with TLS 1.3 but which we're not defining a code point
for.
Then I 'm going to ask you for a challenge. Can you merge two code point about Camellia followings:

TLS_CAMELLIA_128_GCM_SHA256
TLS_CAMELLIA_256_GCM_SHA384